src/sentry/web/frontend/auth_login.py

```diff
@@ -198,14 +198,13 @@ def handle_basic_auth(self, request, **kwargs):
                         om = OrganizationMember.objects.get(
                             organization=organization, email=user.email
                         )
+                        # XXX(jferge): if user is removed / invited but has an acct,
+                        # pop _next so they aren't in infinite redirect on Single Org Mode
                     except OrganizationMember.DoesNotExist:
-                        pass
+                        request.session.pop("_next", None)
```

Comment:
need to do it here too for the "user is removed from org to update 2fa but doesn't have active invite"